olive/cache.py

```diff
@@ -140,7 +140,7 @@ def save_model(
     # save model file/folder
     model_path = model_json["config"]["model_path"]
     if model_path is not None and Path(model_path).exists():
-        if model_json["type"].lower() == "onnxmodel" and not model_json["config"]["is_file"]:
+        if model_json["type"].lower() == "onnxmodel" and model_json["config"]["model_type"] != str(ModelType.LocalFile):
```

Comment:
Nit: no need str()

Reply:
actually, it is needed. since the model_json["config"]["model_type"] is string but ModelType.LocalFile is an enum.

Reply:
If you have `str` type in `class ModelType(str, Enum)`, `ModelType.LocalFile` will be equal to its str value.